code/modules/client/preference_setup/loadout/loadout_suit.dm

```diff
@@ -460,4 +460,9 @@
 /datum/gear/suit/snowsuit/cargo
 	display_name = "snowsuit, supply"
 	path = /obj/item/clothing/suit/storage/snowsuit/cargo
-	allowed_roles = list("Quartermaster","shaft Miner","Cargo Technician","Head of Personnel")
\ No newline at end of file
+	allowed_roles = list("Quartermaster","shaft Miner","Cargo Technician","Head of Personnel")
+
+/datum/gear/suit/ianshirt
```

Comment:
Not really feeling this, for the time being.

Reply:
Poor Ian. Need me to kill the HoP's dreams?

Reply:
Poor Ian. Need me to kill the HoP's dreams?

Reply:
Yes.